From 4f64deb5f1bc7b83fcc4381b7dbbaf71ad4a77c8 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Fri, 7 Jan 2022 01:53:52 +0000
Subject: [PATCH] meson: drop unused argument for i18n.merge_file()

Ignored in Meson < 0.60.0, deprecated since 0.60.1 and fatal since 0.61.0.

panels/applications/meson.build:10:5: ERROR: Function does not take positional arguments.
panels/background/meson.build:10:5: ERROR: Function does not take positional arguments.
panels/camera/meson.build:10:5: ERROR: Function does not take positional arguments.
[...]
---
 panels/applications/meson.build                   | 1 -
 panels/background/meson.build                     | 1 -
 panels/bluetooth/meson.build                      | 1 -
 panels/camera/meson.build                         | 1 -
 panels/color/meson.build                          | 1 -
 panels/datetime/meson.build                       | 2 --
 panels/default-apps/meson.build                   | 1 -
 panels/diagnostics/meson.build                    | 1 -
 panels/display/meson.build                        | 1 -
 panels/info-overview/meson.build                  | 1 -
 panels/keyboard/meson.build                       | 2 --
 panels/location/meson.build                       | 1 -
 panels/lock/meson.build                           | 1 -
 panels/microphone/meson.build                     | 1 -
 panels/mouse/meson.build                          | 1 -
 panels/multitasking/meson.build                   | 1 -
 panels/network/meson.build                        | 1 -
 panels/notifications/meson.build                  | 1 -
 panels/online-accounts/meson.build                | 1 -
 panels/power/meson.build                          | 1 -
 panels/printers/meson.build                       | 1 -
 panels/region/meson.build                         | 1 -
 panels/removable-media/meson.build                | 1 -
 panels/search/meson.build                         | 1 -
 panels/sharing/meson.build                        | 2 --
 panels/sound/meson.build                          | 1 -
 panels/thunderbolt/meson.build                    | 1 -
 panels/universal-access/meson.build               | 1 -
 panels/usage/meson.build                          | 1 -
 panels/user-accounts/meson.build                  | 2 --
 panels/wacom/meson.build                          | 1 -
 panels/wwan/meson.build                           | 1 -
 shell/appdata/meson.build                         | 1 -
 shell/meson.build                                 | 1 -
 tests/interactive-panels/applications/meson.build | 1 -
 35 files changed, 39 deletions(-)

diff --git a/panels/applications/meson.build b/panels/applications/meson.build
index d511bf640..bd4fd6442 100644
--- a/panels/applications/meson.build
+++ b/panels/applications/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-       desktop,
          type : 'desktop',
         input : desktop_in,
        output : desktop,
diff --git a/panels/background/meson.build b/panels/background/meson.build
index eb5e9ec84..ef57c2ef4 100644
--- a/panels/background/meson.build
+++ b/panels/background/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/bluetooth/meson.build b/panels/bluetooth/meson.build
index c3fce7467..4bed812a9 100644
--- a/panels/bluetooth/meson.build
+++ b/panels/bluetooth/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/camera/meson.build b/panels/camera/meson.build
index 4f2611879..bf2e727cf 100644
--- a/panels/camera/meson.build
+++ b/panels/camera/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/color/meson.build b/panels/color/meson.build
index 742caf67b..bb038cd87 100644
--- a/panels/color/meson.build
+++ b/panels/color/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/datetime/meson.build b/panels/datetime/meson.build
index 65683941f..9bbdc7ce0 100644
--- a/panels/datetime/meson.build
+++ b/panels/datetime/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
@@ -20,7 +19,6 @@ i18n.merge_file(
 polkit = 'org.gnome.controlcenter.@0@.policy'.format(cappletname)
 
 i18n.merge_file(
-  polkit,
   input: polkit + '.in',
   output: polkit,
   po_dir: po_dir,
diff --git a/panels/default-apps/meson.build b/panels/default-apps/meson.build
index 68550062d..2a24321a0 100644
--- a/panels/default-apps/meson.build
+++ b/panels/default-apps/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/diagnostics/meson.build b/panels/diagnostics/meson.build
index fac9b4cd8..2f25c2194 100644
--- a/panels/diagnostics/meson.build
+++ b/panels/diagnostics/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/display/meson.build b/panels/display/meson.build
index 12f35cc25..57bdfcfc1 100644
--- a/panels/display/meson.build
+++ b/panels/display/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/info-overview/meson.build b/panels/info-overview/meson.build
index 2fe10b32c..aa81c483f 100644
--- a/panels/info-overview/meson.build
+++ b/panels/info-overview/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/keyboard/meson.build b/panels/keyboard/meson.build
index a2ae0263d..ae71c94b0 100644
--- a/panels/keyboard/meson.build
+++ b/panels/keyboard/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
@@ -45,7 +44,6 @@ xml_files = [
 
 foreach file: xml_files
   i18n.merge_file(
-    file,
     input: file + '.in',
     output: file,
     po_dir: po_dir,
diff --git a/panels/location/meson.build b/panels/location/meson.build
index bfc6e82ad..f4c835d92 100644
--- a/panels/location/meson.build
+++ b/panels/location/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/lock/meson.build b/panels/lock/meson.build
index 0661af4fa..18dbfb2b3 100644
--- a/panels/lock/meson.build
+++ b/panels/lock/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/microphone/meson.build b/panels/microphone/meson.build
index e0847a4c4..bc7f73a20 100644
--- a/panels/microphone/meson.build
+++ b/panels/microphone/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/mouse/meson.build b/panels/mouse/meson.build
index 357aaae81..7b46097f4 100644
--- a/panels/mouse/meson.build
+++ b/panels/mouse/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/multitasking/meson.build b/panels/multitasking/meson.build
index 772b63813..53fd0d8ea 100644
--- a/panels/multitasking/meson.build
+++ b/panels/multitasking/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/network/meson.build b/panels/network/meson.build
index d7cd6e419..43a569092 100644
--- a/panels/network/meson.build
+++ b/panels/network/meson.build
@@ -25,7 +25,6 @@ foreach name: panel_names
   )
 
   i18n.merge_file(
-    desktop,
     type: 'desktop',
     input: desktop_in,
     output: desktop,
diff --git a/panels/notifications/meson.build b/panels/notifications/meson.build
index df702614a..1b04b243d 100644
--- a/panels/notifications/meson.build
+++ b/panels/notifications/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/online-accounts/meson.build b/panels/online-accounts/meson.build
index 82f1d569a..687097724 100644
--- a/panels/online-accounts/meson.build
+++ b/panels/online-accounts/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/power/meson.build b/panels/power/meson.build
index 0e3ea1d56..1cafe2a1b 100644
--- a/panels/power/meson.build
+++ b/panels/power/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/printers/meson.build b/panels/printers/meson.build
index ccce64cb7..042c67acc 100644
--- a/panels/printers/meson.build
+++ b/panels/printers/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/region/meson.build b/panels/region/meson.build
index 68b7f000e..27bc5ff33 100644
--- a/panels/region/meson.build
+++ b/panels/region/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/removable-media/meson.build b/panels/removable-media/meson.build
index b51799af9..fc07c48e5 100644
--- a/panels/removable-media/meson.build
+++ b/panels/removable-media/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/search/meson.build b/panels/search/meson.build
index 39b076bd8..a4dd79d96 100644
--- a/panels/search/meson.build
+++ b/panels/search/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/sharing/meson.build b/panels/sharing/meson.build
index b3803bcda..49dbb0614 100644
--- a/panels/sharing/meson.build
+++ b/panels/sharing/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
@@ -29,7 +28,6 @@ polkit_in = configure_file(
 )
 
 i18n.merge_file(
-  polkit,
   input: polkit_in,
   output: polkit,
   po_dir: po_dir,
diff --git a/panels/sound/meson.build b/panels/sound/meson.build
index fa0f1281f..18f75dd4f 100644
--- a/panels/sound/meson.build
+++ b/panels/sound/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/thunderbolt/meson.build b/panels/thunderbolt/meson.build
index 53aa9b466..38ef294e9 100644
--- a/panels/thunderbolt/meson.build
+++ b/panels/thunderbolt/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/universal-access/meson.build b/panels/universal-access/meson.build
index 5e225fe4b..d845e70a1 100644
--- a/panels/universal-access/meson.build
+++ b/panels/universal-access/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/usage/meson.build b/panels/usage/meson.build
index 74f720ecd..6047ce0ec 100644
--- a/panels/usage/meson.build
+++ b/panels/usage/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/user-accounts/meson.build b/panels/user-accounts/meson.build
index b8ee9d98e..fa56aae02 100644
--- a/panels/user-accounts/meson.build
+++ b/panels/user-accounts/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
@@ -85,7 +84,6 @@ meson.add_install_script('sh', '-c',
 polkit = 'org.gnome.controlcenter.@0@.policy'.format(cappletname)
 
 i18n.merge_file(
-  polkit,
   input: polkit + '.in',
   output: polkit,
   po_dir: po_dir,
diff --git a/panels/wacom/meson.build b/panels/wacom/meson.build
index 5f6222071..88a91377d 100644
--- a/panels/wacom/meson.build
+++ b/panels/wacom/meson.build
@@ -30,7 +30,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-       desktop,
          type : 'desktop',
         input : desktop_in,
        output : desktop,
diff --git a/panels/wwan/meson.build b/panels/wwan/meson.build
index 8c1b02f26..2ab238af4 100644
--- a/panels/wwan/meson.build
+++ b/panels/wwan/meson.build
@@ -11,7 +11,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-       desktop,
          type : 'desktop',
         input : desktop_in,
        output : desktop,
diff --git a/shell/appdata/meson.build b/shell/appdata/meson.build
index 3b36171fb..fdb827969 100644
--- a/shell/appdata/meson.build
+++ b/shell/appdata/meson.build
@@ -1,7 +1,6 @@
 appdata = 'gnome-control-center.appdata.xml'
 
 i18n.merge_file(
-  appdata,
   input: appdata + '.in',
   output: appdata,
   po_dir: po_dir,
diff --git a/shell/meson.build b/shell/meson.build
index 89b96590e..4667804fb 100644
--- a/shell/meson.build
+++ b/shell/meson.build
@@ -23,7 +23,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-       desktop,
          type : 'desktop',
         input : desktop_in,
        output : desktop,
diff --git a/tests/interactive-panels/applications/meson.build b/tests/interactive-panels/applications/meson.build
index 2657f9588..82eed4d5e 100644
--- a/tests/interactive-panels/applications/meson.build
+++ b/tests/interactive-panels/applications/meson.build
@@ -12,7 +12,6 @@ desktop_files = [
 
 foreach desktop_file : desktop_files
   i18n.merge_file(
-              desktop,
                 type : 'desktop',
                input : 'gtp-@0@.desktop.in'.format(desktop_file),
               output : 'gnome-@0@-panel.desktop'.format(desktop_file),
-- 
GitLab

